src/tool/abi/abi_writer.cpp

```diff
@@ -112,7 +112,25 @@ void writer::pop_inline_namespace()
     }
 }
 
-void writer::push_contract_guard(contract_attributes& vers)
+bool writer::push_contract_guard(version ver)
```

Comment:
This return value appears to be unused.

Reply:
It is. You'll just have to look for its use in ternary operators

Reply:
E.g. `w.push_contract_guard(version_from_attribute(fastAttr)) ? 1 : 0`